include/hip/spirv_hip_fp16.h

```diff
@@ -39,8 +39,8 @@ THE SOFTWARE.
 typedef _Float16 _hip_f16;
 typedef _Float16 _hip_f16_2 __attribute__((ext_vector_type(2)));
 #else
-typedef api_half _hip_f16;
-typedef api_half2 _hip_f16_2;
+typedef short _hip_f16;
+typedef short _hip_f16_2 __attribute__((ext_vector_type(2)));
```

Comment:
I was confused about this part.. Where does api_half come from?

Reply:
No idea where it was supposed to come from but it not defined anywhere, AFAIK.